src/deploy/osp_deployer/director.py

```diff
@@ -353,6 +353,9 @@ def node_introspection(self):
                                  "error: {}, stdout: {}".format(exit_status,
                                                                 stderr,
                                                                 stdout))
+        # Turning LLDP off before introspection
+        lldp_off_cmd = "sudo sed -i 's/ipa-collect-lldp=1/ipa-collect-lldp=0/g' /httpboot/inspector.ipxe"
+        self.run(lldp_off_cmd)
```

Comment:
Can you tell me why we need to turn off LLDP prior to running introspection?

Reply:
For OS10 and Mellanox NICs, we were facing LLDP related errors in in-band introspection. Disabling it resolved these issue. We are still investigating this issue that is why we added this temporary workaround.